.CONTRIBUTING.md

```diff
@@ -64,15 +64,43 @@ description: Install the software to compile your decentralized network, configu
 icon: octicons-code-24
 template: index-page.html
 ---
-```
 
+<!-- Add content in markdown here --> 
+
+## Explore What This Section Covers
+
+:::INSERT_GENERATED_CARDS:::
+```
 Some important things to note:
 
 - The `title` represents the `<title>` tag and is used for SEO purposes
 - The `description` represents the meta-description and is also used for SEO purposes
-- The `icon` is used on index pages (`index.md`). So it should be an icon representative of the content on the page. You should stick to using Octicons 24px icons. You can search through the available icons on the [Primer Design System website](https://primer.style/foundations/icons)
-- The `template` defines the template to be used. It should always be `main.html` for new content pages and `index-page.html` when adding a new section
+- The `icon` is used on index pages (`index.md`). It should be an icon representative of the content on the page. You should stick to using Octicons 24px icons, ensuring they are outlined and not filled to maintain style consistency. You can search through the available icons on the [Primer Design System website](https://primer.style/foundations/icons)
 - The `<div>` is populated with links to any pages or subdirectories and is populated automatically by a script at runtime that builds the landing pages
+- The `template` defines the template to be used. It should always `index-page.html` to correctly render the table of contents (TOC)
```

Comment:
What do you mean the table of contents? Because the TOC is usually the navigation element that shows up on the right side of the page, so when I read this that's what I think about and that's incorrect. I think maybe the issue here is just the wrong use of words